src/lib/config/metadata/metadatalist.go

```diff
@@ -15,7 +15,6 @@
 package metadata
 
 import "github.com/goharbor/harbor/src/common"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380